### PR TITLE
docs: updated CHATNVIDIA notebooks

### PR DESCRIPTION
**Pull request by dglogo**
Updated notebook for tool calling support in chat models

### CODE REVIEW

**Review by efriis — COMMENTED** [review]
Hey! Can we update this to not use the legacy ConversationChain? Here's the migration guide! https://python.langchain.com/v0.2/docs/how_to/migrate_chains/#conversationchain

File: docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb [hunk 1]
```diff
@@ -593,38 +493,24 @@
...
-    "conversation = RunnableWithMessageHistory(\n",
-    "    chat,\n",
-    "    get_session_history,\n",
-    ")\n",
-    "\n",
-    "conversation.invoke(\n",
-    "    \"Hi I'm Srijan Dubey.\",  # input or query\n",
-    "    config=config,\n",
-    ")"
+    "conversation = ConversationChain(llm=chat, memory=ConversationBufferMemory())"
```
Comment (ccurme) [inline docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb:496]: `ConversationChain` is deprecated in favor of `RunnableWithMessageHistory`. See [API reference](https://api.python.langchain.com/en/latest/chains/langchain.chains.conversation.base.ConversationChain.html). Why make this change?

File: docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb [hunk 2]
```diff
@@ -447,100 +444,14 @@
     "llm.invoke(f'What\\'s in this image?\\n<img src=\"{base64_with_mime_type}\" />')"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "id": "3e61d868",
-   "metadata": {},
-   "source": [
-    "#### **Advanced Use Case:** Forcing Payload \n",
```
Comment (ccurme) [inline docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb:455]: Why remove this?

File: docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb [hunk 3]
```diff
@@ -302,9 +302,6 @@
     "\n",
     "NVIDIA also supports multimodal inputs, meaning you can provide both images and text for the model to reason over. An example model supporting multimodal inputs is `nvidia/neva-22b`.\n",
     "\n",
-    "\n",
-    "These models accept LangChain's standard image formats, and accept `labels`, similar to the Steering LLMs above. In addition to `creativity`, `complexity`, and `verbosity`, these models support a `quality` toggle.\n",
```
Comment (ccurme) [inline docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb:306]: Why remove this?

**Review by ccurme — CHANGES_REQUESTED** [review]
Looks good. Left some questions. Only blocking concern is replacement of `RunnableWithMessageHistory` with `ConversationChain`.

File: docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb (hunk 2 above)
Comment (dglogo) [reply docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb:455]: Not officially supported

File: docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb (hunk 3 above)
Comment (dglogo) [reply docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb:306]: unsupported

File: docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb (hunk 1 above)
Comment (dglogo) [reply docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb]: updated!

**Comment by dglogo** [comment]
> Looks good. Left some questions. Only blocking concern is replacement of `RunnableWithMessageHistory` with `ConversationChain`.

Updated!

**Review by ccurme — APPROVED** [review]

**Comment by ccurme** [comment]
> Updated!

Thank you for iterating!

**Pull request merged**